### DISCUSSION TITLE
G6的流水线pipe布局的v5版本什么时间发布，版本分支是什么？

### DISCUSSION THREAD

**Discussion by kel66** [community/community#135623] [Programming Help]
### Body

想请问下g6的流水线pipe布局在v5中发布了吗? 如果待发布，可否告知下计划发布的版本分支和时间计划? 语雀社区的待办列表显示，pipeline布局目前处于社区已认领的状态，会根据(多次)CR建议进行修改，等待合并入v5分支。
详见：
1、https://www.yuque.com/antv/g6/qlew63zmylpsifi1
2、https://github.com/antvis/G6/issues/4933

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by ebndev** [comment]
Hi @kel66, thanks for participating!

Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting to ensure discussions are adhering to our [Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md). 

We’ll be locking this post for now, but please feel encouraged to create a new discussion in English if you still need support.
